cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/media/class-filter.php

```diff
@@ -378,6 +378,8 @@ public function filter_out_local( $content ) {
 			}
 
 			$content = str_replace( $asset, $new_tag, $content );
```

Comment:
This could be simplified to

```suggestion
			$content = str_replace( array( $asset, $url ), array( $new_tag, $cloudinary_url ), $content );
```